src/components/calcite-color/calcite-color.tsx

```diff
@@ -80,9 +80,20 @@ export class CalciteColor {
       return;
     }
 
-    const value = this.toValue(color);
+    this.value = this.toValue(color);
+  }
+
+  /**
+   * The format of the value property.
+   *
+   * When "auto", the format will be inferred from `value` when set.
+   */
+  @Prop() format: "auto" | SupportedMode = "auto";
```

Comment:
can you add `auto` to the supportedmode type?

Reply:
I'll refactor to create a `Format` type. `SupportedMode` is internal.